### PR TITLE
feat: integrate Operator API with the UI

### LINKED ISSUE #1438: Integrate Operator API
This issue is being created to help keep track of the UI work that has been completed and needs to be integrated with the Quartz API.

### PR DESCRIPTION
**Pull request by Asalem1**
Closes #1438 

This PR integrates the operator API from Quartz with the UI and added cypress tests for the newly integrated unification work. 

operator

### CODE REVIEW

File: cypress/e2e/cloud/checkout.test.ts [hunk 1]
```diff
@@ -40,7 +40,7 @@ describe('Checkout Page', () => {
     })
   })
 
-  it('should render Set My Limits', () => {
+  it('should render the checkout page and allow for pointing and clicking', () => {
```
Comment (Asalem1) [inline cypress/e2e/cloud/checkout.test.ts:43]: changed for accuracy

File: src/operator/account/RemoveFromAccount.tsx [hunk 2]
```diff
@@ -13,6 +13,7 @@ const RemoveFromAccount: FC<Props> = ({removeUser}) => {
         confirmationLabel="This will not delete a user. It will remove them from their account and org allowing them to join a new organization. Are you sure?"
         onConfirm={removeUser}
         text="Remove User"
+        testID="remove-user"
```
Comment (Asalem1) [inline src/operator/account/RemoveFromAccount.tsx:16]: super weird, the confirmation button was appending an `--button` onto this?

**Comment by ui-ci-bot-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 46 | :red_circle:
accessibility | 72 | :yellow_circle:
seo | 67 | :yellow_circle:
pwa | null | :hankey:
best-practices | 75 | :yellow_circle:


[Compare your scores with master branch](https://influxdb.aws.influxdata.io/orgs/844910ece80be8bc/dashboards/073c4b95f1089000?lower=now%28%29%20-%202d&vars%5Blighthouse_score_branch%5D=master)

File: cypress/e2e/cloud/operator.test.ts [hunk 3]
```diff
@@ -0,0 +1,268 @@
...
+          }).then(() => {
+            cy.visit(`/operator`)
+            cy.getByTestID('operator-page--title').contains('2.0 Resources')
+          })
+        })
+      })
+    })
+  })
+
+  it('should render the Operator page and allow for RUD operations', () => {
```
Comment (genehynson) [inline cypress/e2e/cloud/operator.test.ts:20]: `RUD` -> `CRUD`

File: cypress/e2e/cloud/operator.test.ts [hunk 4]
```diff
@@ -0,0 +1,268 @@
...
+    cy.getByTestID('table-body').within(() => {
+      cy.getByTestID('table-row').should('have.length', 1)
+    })
+
+    // Make sure that the popover closes when clicking out
+    cy.getByTestID('refless-popover--contents').should('not.exist')
+
+    cy.getByTestID('operator-resource--searchbar')
+      .clear()
+      .type('salt_bae', {force: true, delay: 50})
```
Comment (genehynson) [inline cypress/e2e/cloud/operator.test.ts:63]: 😆

File: cypress/e2e/cloud/operator.test.ts [hunk 5]
```diff
@@ -0,0 +1,268 @@
...
+      force: true,
+      delay: 50,
+    })
+    cy.getByTestID('table-body').within(() => {
+      cy.getByTestID('table-row').should('have.length', 1)
+    })
+
+    cy.getByTestID('operator-resource--searchbar')
+      .clear()
+      .type('invalid', {force: true, delay: 50})
```
Comment (genehynson) [inline cypress/e2e/cloud/operator.test.ts:108]: just curious what the `delay: 50` is for in all of these?

File: src/operator/account/AccountBillingContact.tsx [hunk 6]
```diff
@@ -31,14 +31,14 @@ const AccountBillingContact: FC = () => {
   }
 
   return (
-    <div>
+    <>
```
Comment (genehynson) [inline src/operator/account/AccountBillingContact.tsx:34]: noice

File: src/operator/account/RemoveFromAccount.tsx (hunk 2 above)
Comment (genehynson) [reply src/operator/account/RemoveFromAccount.tsx:16]: 😭  https://github.com/influxdata/clockface/blob/master/src/Components/Button/Composed/ConfirmationButton.tsx#L142

**Review by genehynson — APPROVED** [review]
WOOO!!!!

**Review by genehynson — APPROVED** [review]
another approve cuz gif

**Comment by genehynson** [comment]
> Metric	Score	Level
> performance	46	🔴
> accessibility	72	🟡
> seo	67	🟡
> pwa	null	💩
> best-practices	75	🟡
> [Compare your scores with master branch](https://influxdb.aws.influxdata.io/orgs/844910ece80be8bc/dashboards/073c4b95f1089000?lower=now%28%29%20-%202d&vars%5Blighthouse_score_branch%5D=master)

don't worry about this - your scores are fine. https://app.circleci.com/pipelines/github/influxdata/monitor-ci/2512/workflows/b4c1dc89-2ffe-4f23-a2b3-4b335bbc4ffb/jobs/22127

this is happening because of this bug: https://github.com/influxdata/monitor-ci/issues/274

File: cypress/e2e/cloud/operator.test.ts (hunk 5 above)
Comment (Asalem1) [reply cypress/e2e/cloud/operator.test.ts:108]: omg so FOR SOME REASON when I would just set the `type('whatever')` most of the text would get trimmed. I tried a million different things (cy.focus, cy.should().type(), even a cy.wait thinking that the issue was that the DOM hadn't loaded) but in the end this is what I came from. I _think_ it has to do with the fact that we're debouncing the search text but i'm not :100: on that

Comment (genehynson) [reply cypress/e2e/cloud/operator.test.ts:108]: ohhhhh yeah the debounce thing actually makes sense. It probably confuses cypress into thinking it's done or something. Cool, nice fix!

File: cypress/e2e/cloud/operator.test.ts (hunk 3 above)
Comment (Asalem1) [reply cypress/e2e/cloud/operator.test.ts:20]: Nothing is being created

**Pull request merged**